src/Symfony/Component/Mime/MimeTypes.php

```diff
@@ -51,7 +51,8 @@ public function __construct(array $map = [])
             $this->extensions[$mimeType] = $extensions;
 
             foreach ($extensions as $extension) {
-                $this->mimeTypes[$extension] = $mimeType;
+                $this->mimeTypes[$extension] = $this->mimeTypes[$extension] ?? [];
+                array_push($this->mimeTypes[$extension], $mimeType);
```

Comment:
You can shorten the patch a bit by using `[]`:

```php
$this->mimeTypes[$extension][] = $mimeType;
```

Reply:
done.